README.md

```diff
@@ -1,2 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # AppleClone
+
 Old Apple Website Clone project for Microverse program for practicing backgrounds and gradients. Project by Helcio Andre and Alan Solis
+
+
+![screenshot](img/screenshot.jpg)
+
+## Sample of Old Apple Website Clone with the following features:
+
+- Navigation Bar with gradients
+- Backgrounds features on all the project
+- Flexbox
+
+
+## Built With
+
+- HTML
+- CSS/SCSS
+
+
```

Comment:
[OPTIONAL]
You can add SASS here, remember to add all the techs you use in the readme to highlight it